src/random.jl

```diff
@@ -0,0 +1,80 @@
+#=
+Generate random NormalFormGame instances.
+=#
+
+#
+# Random Games Generating
+#
+"""
+    random_game{N}(nums_actions::NTuple{N,Int})
+
+Return a random NormalFormGame instance where the payoffs are drawn
+independently from the uniform distribution on [0, 1).
+
+# Arguements
+* `nums_actions::NTuple{N,Int}`: Tuple of the numbers of actions,
+    one for each player.
```

Comment:
I don't know if this makes a difference, but shouldn't it be

```
* `nums_actions::NTuple{N,Int}`: Tuple of the numbers of actions,
  one for each player.
```

rather than

```
* `nums_actions::NTuple{N,Int}`: Tuple of the numbers of actions,
    one for each player.
```